dotCMS/src/integration-test/java/com/dotcms/publishing/job/SiteSearchJobImplTest.java

```diff
@@ -560,5 +700,32 @@ public void test_MultilangContent_IndexingAllLanguages()
         }
     }
 
+    private HTMLPageAsset createHtmlPageAsset(Language lang1, Contentlet contentlet)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'lang1' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5303548)